Mixpanel/MPNotification.m

```diff
@@ -86,11 +95,27 @@ - (instancetype)initWithJSONObject:(NSDictionary *)object {
         _backgroundColor = backgroundColor.unsignedIntegerValue;
         _imageURL = imageURL;
         _image = nil;
+        _displayTriggers = parsedDisplayTriggers;
     }
 
     return self;
 }
 
+- (BOOL)hasDisplayTriggers {
+    return _displayTriggers != nil && [_displayTriggers count] > 0;
```

Comment:
ObjC Coding style: prefer `self.displayTriggers` than `_displayTriggers` except in init. the same comment to all occurrences in this PR.